engine/src/main/scala/cromwell/engine/backend/jes/JesBackend.scala

```diff
@@ -190,14 +190,51 @@ object JesBackend {
   def jesLogFilename(key: BackendCallKey) = s"${jesLogBasename(key)}.log"
   def jesReturnCodeFilename(key: BackendCallKey) = s"${jesLogBasename(key)}-rc.txt"
   def globDirectory(glob: String) = s"glob-${glob.md5Sum}/"
-  def globOutputPath(callPath: Path, glob: String) = callPath.resolve(s"glob-${glob.md5Sum}/")
+  def rootPath(options: WorkflowOptions): String = options.getOrElse(GcsRootOptionKey, ProductionJesConfiguration.jesConf.executionBucket).stripSuffix("/")
+
+  implicit class JesEnhancedJobDescriptor(val jobDescriptor: BackendCallJobDescriptor) extends AnyVal {
```

Comment:
I was thinking about putting this in a jes package object instead of adding to the already giant JesBackend object